dotCMS/src/main/java/com/dotcms/rendering/velocity/services/PageRenderUtil.java

```diff
@@ -81,7 +74,7 @@
     private final TagAPI        tagAPI        = APILocator.getTagAPI();
     private final PersonaAPI    personaAPI    = APILocator.getPersonaAPI();
 
-    final IHTMLPage htmlPage;
+    final HTMLPageAsset htmlPage;
```

Comment:
Not sure why use the implementation rather the interface

Reply:
because I need get the site here 

https://github.com/dotCMS/core/pull/18033/files#diffc7538880be24298c9c76f10f1c8efaffR112

Reply:
done https://github.com/dotCMS/core/pull/18033/commits/286b9c09f52d6e0bb78ec9761ec3000c06d7f38a#diff-c7538880be24298c9c76f10f1c8efaffR92

https://github.com/dotCMS/core/pull/18033/commits/286b9c09f52d6e0bb78ec9761ec3000c06d7f38a#diff-2c7c10f84439ab07f53fa4dda91df2dcR23